<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FdWidget</class>
 <widget class="QWidget" name="FdWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>618</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Scaffold</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Node:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="refitMethod">
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>MinOBB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AABB</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="addLink">
        <property name="text">
         <string>Add Link</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="fitCuboid">
        <property name="text">
         <string>Fit</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="changeCuboidType">
        <property name="text">
         <string>Change Type</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3" colspan="2">
       <widget class="QPushButton" name="loadScaffold">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QPushButton" name="saveScaffold">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Link:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Graph:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QPushButton" name="createScaffold">
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="fitMethod">
        <item>
         <property name="text">
          <string>AABB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MinOBB</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="3">
       <widget class="QLabel" name="fdPath">
        <property name="text">
         <string>this is a place holder...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Visualization</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QCheckBox" name="showCuboids">
        <property name="text">
         <string>Cuboids</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="showScaffold">
        <property name="text">
         <string>Scaffold</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="showMesh">
        <property name="text">
         <string>Mesh</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QCheckBox" name="showAABB">
        <property name="text">
         <string>AABB</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="showFolded">
        <property name="text">
         <string>Show Folded</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="1">
    <widget class="QPushButton" name="test">
     <property name="text">
      <string>TEST</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Foldabilizer</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="2" colspan="2">
       <widget class="QPushButton" name="createLayers">
        <property name="text">
         <string>Create Layers</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QListWidget" name="layerList"/>
      </item>
      <item row="1" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Scaffolds</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QListWidget" name="chainList"/>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Target Volume</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QDoubleSpinBox" name="targetV">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QListWidget" name="scaffoldList"/>
      </item>
      <item row="0" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Direction:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="pushDirection">
        <item>
         <property name="text">
          <string>X</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Z</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ALL</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Layers</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Chains</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QPushButton" name="fold">
        <property name="text">
         <string>Fold</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="foldLayer">
        <property name="text">
         <string>Fold Layer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
